src/main.cpp

```diff
@@ -5311,7 +5310,7 @@ bool GetEarliestStakeTime(std::string grcaddress, std::string cpid)
                         {
 						    myCPID = pblockindex->GetCPID();
                         }
-                        if (cpid == myCPID && nCPIDTime==0 && myCPID != "INVESTOR")
+                        if (cpid == myCPID && nCPIDTime==0 && !IsResearcher(myCPID))
```

Comment:
This one should check for `true`, right?

Reply:
your right. always something that slips by thats why we review :)